index.js

```diff
@@ -44,7 +46,7 @@ module.exports = function(bytes, options){
   if (bytes === 0) {
     value = 0;
     unit = config.units.B;
-    return sprintf(config.outputFormat, value, unit);
+    return sprintf(config.outputFormat, numberFormat.apply( numberFormat, [value].concat(config.numberFormatOptions)), unit);
```

Comment:
Actually can you make the if statement above into an if-else and kill this line? I want to get rid of the two return statements in this method because they're duplicating code. Should look something like:

```
  if (bytes === 0) {
    value = 0;
    unit = config.units.B;
  } else {
    var converted = convert(bytes, config.units, 'B', config.convert.to, config.decimal);
    value = converted.value;
    unit = converted.unit;

    if (config.minSigFigs && value.toString().replace('.', '').length <= config.minSigFigs){
      value = value.toPrecision(config.minSigFigs);
    } else if (value > 1 || unit !== 'B') {
      value = value.toFixed(config.digits);
    }
  }
```